### ISSUE TITLE
Problems with Pattern Search Window(GUI related)

### ISSUE THREAD

**Issue by gmastergreatee** [x64dbg/x64dbg#1755]
Debugger Version : Oct 10, 2017
OS : Win 8 x64
Resolution : 1920 x 1080

### Description
As shown in the following image, there must not be 2 vertical scroll bars on __Hex__ textbox.
Moreover, after entering a lot of bytes in the hexbox, I'm unable to scroll down.

![asd](https://user-images.githubusercontent.com/17880848/31444163-07fd4cda-aeb9-11e7-8fda-8d68def8d48c.png)

### Feature Request
Also display the current size of bytes entered.
It may be displayed alongside the __"Keep Size"__ checkbox, viz :-
```
/// [checkbox] Keep Size ({Current Size})
[] Keep Size (87)
```
I decided to merge the request here because it seemed easy to fix.

**Comment by mrexodia** [comment]
Display scaling: 125%

**Comment by gmastergreatee** [comment]
Two vertical scrollbars are a result of Display scaling(being set to 125%) but the vertical scrollbar still cannot be used to scroll down if excessive bytes longer than that could be held in the hexbox are entered.

**Comment by gmastergreatee** [comment]
Fixed in #1941
